packages/vue-app/template/client.js

```diff
@@ -36,6 +37,11 @@ const NUXT = window.<%= globals.context %> || {}
 
 Object.assign(Vue.config, <%= serialize(vue.config) %>)<%= isTest ? '// eslint-disable-line' : '' %>
 
+<% if (isDev) { %>
+const logger = consola.withScope('nuxt:ssr')
+const logs = NUXT.logs || []
+logs.forEach(logObj => logger[logObj.type](logObj))
```

Comment:
`logs` can be deleted after logging to console.

Reply:
Added, thanks :)